lib/ansible/modules/web_infrastructure/ansible_tower/tower_project.py

```diff
@@ -82,6 +97,17 @@
     organization: "test"
     state: present
     tower_config_file: "~/tower_cli.cfg"
+
+- name: Add Tower Project with cache timeout and custom virtualenv
+  tower_project:
+    name: "Foo"
+    description: "Foo bar project"
+    organization: "test"
+    scm_update_on_launch: True
+    scm_update_cache_timeout: 60
+    custom_virtualenv: "/var/lib/awx/venv/ansible-2.2"
```

Comment:
not sure if this is correct documentation, let me check against my server...

Reply:
no, never mind, this is correct. This field takes the absolute path including the venv root, so this is good documentation.